### DISCUSSION TITLE
TameMyCerts windows event number 6

### DISCUSSION THREAD

**Discussion by ldc-2024** [Sleepw4lker/TameMyCerts#32] [General]
Hi,
TameMyCerts windows event number 6 provides useful information about why a certificate was denied.
Is there a way to show the description of that event to the requestor rather in a pop-up window or in command line?

**Comment by Sleepw4lker** [comment]
Hi @ldc-2024 we are somewhat limited in what we can return to the CA process - which in turn is what gets returned to a client via MS-WCCE. It boils down to the codes defined in winerror.h. The detailed information that you see in the server logs cannot be passed to the client directly.
